### PR TITLE
Updated end-user license agreement

### PR DESCRIPTION
**Pull request by Al3LawsRobotics**
Received updated EULA from attorneys.

### CODE REVIEW

File: docs/sources/copyrights.rst [hunk 1]
```diff
@@ -1,47 +1,81 @@
...
+1.3 3Laws reserves the right to change or modify portions of this Agreement at any time.  If 3Laws does so, it will post the changes on this page and will indicate at the top of this page the date this Agreement was last revised. 3Laws will also notify Customer, either through the Software user interface, in an email notification or through other reasonable means. Any such changes will become effective no earlier than fourteen (14) days after being posted, except that changes addressing new functions of the Software or changes made for legal reasons may become effective immediately. Customer’s continued use of the Software after the date any such changes become effective constitutes acceptance of the new Agreement.
```
Comment (PierreQuebriac) [inline docs/sources/copyrights.rst:15]: We should add the date of last revise at the top of the doc according to this clause

File: docs/sources/copyrights.rst [hunk 2]
```diff
@@ -1,47 +1,81 @@
...
+2.3 Customer acknowledges that the Software (i) records and logs certain usage information related to the Software and related robotic systems, and (ii) requires license keys from 3Laws in order to function.  Customer shall not disable or tamper with such recording, logging and/or license key functionality in any way and shall provide 3Laws with reasonable access (either direct or remote, at 3Laws’ option) to all systems on which the Software is installed, operated or used for purposes of monitoring or retrieving any such information or license keys and otherwise cooperate with 3Laws’ efforts to obtain and monitor such information and 3Laws license keys.
```
Comment (PierreQuebriac) [inline docs/sources/copyrights.rst:25]: Is it standard ? Seems that we ask for full access to their system

**Review by PierreQuebriac — APPROVED** [review]

File: docs/sources/copyrights.rst [hunk 3]
```diff
@@ -1,47 +1,81 @@
...
-##########
+Copyrights and Licenses
+#######################
 
 The 3Laws Robotics Supervisor is a proprietary, commercial package. This is the 3Laws End User License Agreement
 
-END-USER LICENSE AGREEMENT
-NOTICE:  READ THIS BEFORE USING THIS PROGRAM
+3LAWS SOFTWARE LICENSE AGREEMENT
+********************************
```
Comment (PierreQuebriac) [inline docs/sources/copyrights.rst:8]: Adding date here according to 1.3 ?

**Review by PierreQuebriac — CHANGES_REQUESTED** [review]

File: docs/sources/copyrights.rst (hunk 2 above)
Comment (Al3LawsRobotics) [reply docs/sources/copyrights.rst:25]: I brought this issue up with Amir.

**Pull request merged**